google-api/googleapis.js

```diff
@@ -0,0 +1,158 @@
+        description: 'A weekly meet up (with Design team) and product team members who are interested to discuss implement and roll out design centric planning and product development.',
+        location: 'Toronto-1-Large Board  Room (3)',
+        creator: { email: 'do.park@eqworks.com' },
+        organizer: { email: 'do.park@eqworks.com' },
+        start: {
+          dateTime: '2020-12-21T11:00:00-05:00',
+          timeZone: 'America/Toronto'
+        },
+        end: {
+          dateTime: '2020-12-21T12:00:00-05:00',
+          timeZone: 'America/Toronto'
+        },
+        recurringEventId: '1itgg4cctkv3btud9tqntqe5ul_R20200914T150000',
+        originalStartTime: {
+          dateTime: '2020-12-21T11:00:00-05:00',
+          timeZone: 'America/Toronto'
+        },
+        iCalUID: '1itgg4cctkv3btud9tqntqe5ul_R20200914T150000@google.com',
+        sequence: 2,
+        attendees: [
+          [Object], [Object],
+          [Object], [Object],
+          [Object], [Object],
+          [Object], [Object],
+          [Object], [Object],
+          [Object], [Object],
+          [Object]
+        ],
+        extendedProperties: { shared: [Object] },
+        conferenceData: {
+          entryPoints: [Array],
+          conferenceSolution: [Object],
+          conferenceId: '81722919688',
+          signature: 'AL9oL6WI1XjFbxNdjcnpxw6hy+nb',
+          notes: 'Passcode: 719300',
+          parameters: [Object]
+        },
+        reminders: { useDefault: true }
+      }, ... ]
+   */
+
+    const events = items.map(({
+      summary,
+      start: { dateTime: startDate },
+      end: { dateTime: endDate },
+      htmlLink,
+    }) => ({
+      summary,
+      timeSloth: [
+        'between',
+        new Date(startDate).toLocaleTimeString(),
+        'and',
+        new Date(endDate).toLocaleTimeString()
+      ].join(' '),
+      htmlLink,
+    }))
+
+    return events.length
+      ? { day: new Date(start).toDateString(), link, events }
+      : null
+  } catch(error) {
+    console.error(error)
+  }
+}
+
+module.exports.gCalendarCreateEvent = async ({date, start: _start, end: _end}) => {
+/** https://developers.google.com/calendar/v3/reference/events/insert#javascript */
+// const gCalendarCreateEvent = async () => {
+  const calendar = google.calendar({
+    version: 'v3',
+    auth: oauth2Client
+  })
+
+  const start = new Date(`${date} ${_start}`).toISOString()
+  const end = new Date(`${date} ${_end}`).toISOString()
+
+  try {
+    const event = {
+      summary: 'Demo',
+      start: {
+        // 'dateTime': '2020-12-18T09:00:00',
+        'dateTime': start,
+        'timeZone': 'America/Toronto'
+      },
+      end: {
+        // 'dateTime': '2020-12-18T17:00:00',
+        'dateTime': end,
+        'timeZone': 'America/Toronto'
+      },
+    }
+    const { status } = await calendar.events.insert({
+      calendarId,
+      resource: event,
+    })
+    return status
+  } catch (err) {
+    console.error( err)
+  }
+}
```

Comment:
a couple of recommendations.

there's no imperative logic involved to warrant the use of `async/await` (unless strongly preferred).

```js
module.exports.gCalendarCreateEvent = ({date, start: _start, end: _end}) => {
  const calendar = ...
  const start = ...
  const end = ...
  const event = ...
  return calendar.events.insert({ ... }).then(({ status }) => status)
}
```

this adds to the advantage of more explicit error propagation to be handled altogether at `/interactive` handler, where it's invoked, with other uncontrolled promises involved (the axios.post action)